src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,12 @@
+Manifest-Version: 1.0
+Built-By: jan
+Class-Path: lucene-core-4.9.0.jar lucene-queries-4.9.0.jar lucene-quer
+ yparser-4.9.0.jar lucene-sandbox-4.9.0.jar lucene-analyzers-common-4.
+ 9.0.jar je-18.1.11.jar poi-3.10.1.jar commons-codec-1.5.jar poi-excel
+ ant-3.10.1.jar poi-ooxml-3.10.1.jar poi-ooxml-schemas-3.10.1.jar xmlb
+ eans-2.6.0.jar stax-api-1.0.1.jar dom4j-1.6.1.jar xml-apis-1.0.b2.jar
+  ant-1.8.2.jar ant-launcher-1.8.2.jar
+Created-By: Apache Maven 3.6.3
+Build-Jdk: 1.8.0_382
+Main-Class: org.ohdsi.usagi.Launcher
+

```

Comment:
It seems that this should be automatically added to `target`, not to `src`? Or at least it should probably not be committed?

Reply:
Well... where do I begin? It may have not been a good idea to actually use maven-shade-plugin, since I only created this file to be able to suppress some warnings about multiple instances of MANIFEST.MF (from various dependencies). So instead of feeling lucky that maven-shade-plugin happens to choose the right one, I made this one the right one explicitly. And I would feel OK with that if it wasn't for the Class-Path (oops, I did not notice that). I will review, possible refactor and let you know/discuss the outcome.

Reply:
Ok, solved it.